bundles/net.tourbook.device/src/net/tourbook/device/sporttracks/FitLogSAXHandler.java

```diff
@@ -161,54 +201,110 @@
       _weatherId.put("Thunder",        IWeather.WEATHER_ID_LIGHTNING); //                 //$NON-NLS-1$
       _weatherId.put("Snow",           IWeather.WEATHER_ID_SNOW); //                      //$NON-NLS-1$
       _weatherId.put("Haze",           IWeather.WEATHER_ID_PART_CLOUDS); //         *)    //$NON-NLS-1$
+      _weatherId.put("Mist",           IWeather.WEATHER_ID_PART_CLOUDS); //         *)    //$NON-NLS-1$
 // SET_FORMATTING_ON
    }
 
    private class Activity {
 
-      private ArrayList<TimeData>  timeSlices         = new ArrayList<>();
-      private ArrayList<Lap>       laps               = new ArrayList<>();
-      private ArrayList<Pause>     pauses             = new ArrayList<>();
-      private ArrayList<Equipment> equipmentNames     = new ArrayList<>();
+      private ArrayList<TimeData>                 _timeSlices             = new ArrayList<>();
```

Comment:
In my new code, I try to follow this [rule](https://rules.sonarsource.com/java/RSPEC-1319) whenever possible

Reply:
> In my new code, I try to follow this [rule](https://rules.sonarsource.com/java/RSPEC-1319) whenever possible

ok I'll also try to follow that rule and this case for instance use "List" as  type instead of ArrayList.